platform-hub-api/app/serializers/user_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class UserSerializer < BaseSerializer
-  attributes :id, :name, :email, :role, :last_seen_at
+  attributes :id, :name, :email, :role, :last_seen_at, :is_active
```

Comment:
Having said that, maybe we still want to show the active/inactive state of users to other users, e.g. in the project team listing? (in which case we may need a bit of UI for that)